libs/common/src/helpers/payload-verification/is-verified-payload-by-type.test.ts

```diff
@@ -18,26 +15,21 @@ describe('isVerifiedPayloadByType', () => {
       template: templates.nameAndAge,
       description: 'when payload and template have the same properties and data types',
     },
-    {
-      payload: { name: 'First', address: { city: 'London', country: 'UK' } },
-      template: templates.nameAndAddress,
-      description: 'when payload and template have nested objects with same properties and data types',
-    },
     {
       payload: { name: 'First', address: { city: 'London', country: 'UK' } },
       template: templates.nameAndAddressGeneric,
       description: 'when payload and template have nested objects with the template checking for a generic object',
     },
-    {
-      payload: { items: [1, 2, 3] },
-      template: templates.items,
-      description: 'when payload and template have arrays with same data types',
-    },
     {
       payload: { items: [1, 2, 3] },
       template: templates.itemsGeneric,
       description: 'when payload and template have arrays with the template checking for a generic array',
     },
+    {
+      payload: { items: null },
+      template: templates.itemsNull,
+      description: 'when field is an object in template but null in payload',
```

Comment:
Think this should be 
```suggestion
      description: 'when payload and object are null ',
```